cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/cube/DefaultCube.java

```diff
@@ -65,17 +65,20 @@
 
   private final Map<Integer, FactTable> resolutionToFactTable;
   private final Map<String, ? extends Aggregation> aggregations;
+  private final Map<String, AggregationAlias> aggregationAliasMap;
```

Comment:
i am not replacing the name in getDimensionName list. it stays the same, which is "run" tag, however i want to assign the value of "wfr" (workflow run) to the "run" tag. the value we get from facts.